smuggler/utils.py

```diff
@@ -30,15 +33,21 @@ def serialize_to_response(app_labels=None, exclude=None, response=None,
     error_stream = StringIO()
     dumpdata = DumpData()
     dumpdata.style = no_style()
-    dumpdata.execute(*app_labels, **{
+    args = {
```

Comment:
I would name it `kwargs` instead `args`. The same applies at the similar change below.